modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -46,9 +46,11 @@ baremetal   4.10.12   True        False         False      3d15h
 +
 [source,terminal]
 ----
-$ oc delete bmh -n openshift-machine-api vmaster-0
-$ oc delete machine -n openshift-machine-api kni1-master-0
+$ oc delete bmh -n openshift-machine-api <host_name>
+$ oc delete machine -n openshift-machine-api <machine_name>
 ----
++ 
+Replace `<host name>` with the name of the host and `machine name` with the name of the machine. The machine name appears under the `CONSUMER` field
```

Comment:
Please change `<host name>` to `<host_name>`  and  `machine name` to `<machine_name>`

Reply:
Oops... thanks for spotting. I fixed this now.